themes.txt

```diff
@@ -189,7 +189,7 @@ github.com/koirand/pulp
 github.com/kongdivin/hugo-theme-okayish-blog
 github.com/kritoke/darksimplicity
 github.com/lasseborly/anybodyhome
-github.com/Lednerb/bilberry-hugo-theme
+github.com/Lednerb/bilberry-hugo-theme/v3
```

Comment:
@igor-baiborodine Could you please explain this change? AFAIK there is no such schema <user>/<repo>/v3 on GitHub. What does this means?

Reply:
@sergeyklay, it's related to PR https://github.com/gohugoio/hugoThemesSiteBuilder/pull/157:
![image](https://user-images.githubusercontent.com/1027701/181017619-6b71f69c-4e3c-4a34-8759-7da8efc4c27b.png)

![image](https://user-images.githubusercontent.com/1027701/181017974-00206875-595d-4717-b637-67205848a166.png)

In the Bilberry Hugo theme, we had an issue with versioning when migrating to Hugo modules. See this issue https://github.com/Lednerb/bilberry-hugo-theme/issues/344 to understand better why the path ends with `v3`.

Reply:
Hm... interesting. Thank you!